app/src/main/java/android/coding/interview/makeitawesome/adapter/PhotosAdapter.java

```diff
@@ -35,11 +64,34 @@ public int getItemCount() {
         return mData.size();
     }
 
-    public static class ViewHolder extends RecyclerView.ViewHolder {
-        final TextView mText;
+    public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
:(
I don't know. But I'd say he removes the static because it disturbs him in order to access an outer class element (the clickListener). That's not so good...